sites/all/modules/reol_app_feeds/src/Helper/NodeHelper.php

```diff
@@ -230,7 +230,7 @@ public function loadReferences($entity, $field_name, $multiple = TRUE) {
     }
     $values = $entity->{$field_name}[LANGUAGE_NONE];
     $nids = array_column($values, 'target_id');
-    $nodes = node_load_multiple($nids);
+    $nodes = $this->loadNodes($nids);
```

Comment:
`entity_load` has an `$conditions` array that should be used, where the condition could be node status.